styles.css

```diff
@@ -79,7 +79,7 @@
 }
 
 .headline {
-  background-image: url("header-shapes mobile.svg");
+  background-image: url("./image/header-shapes\ mobile.svg");
   background-repeat: no-repeat;
   display: flex;
   flex-direction: column;
```

Comment:
- In your headline section, you currently do not have any images folder nor `header-shapes` files. This is why your background images are not being referenced.

- Can we kindly download the images and create a folder for them in our vscode, that way, you can reference the image as a background in your portfolio. 🤗 

- Can we ensure to remove the border around your `Let's Connect` button ?

You can add the code snippet i have written out below to your CSS for the button, but note that it will affect all buttons that will be added in subsequent milestones for your portfolio. So maybe you can add a class to the button and style it. Or what do you think??
```
button {
  border: none;
  background: none;
}
```


- Also, it is important to stick to the colors used on Figma as much as possible. Kindly compare the color of your `hamburger menu` and your `social icons` to that on Figma. They are quite different. Can we fix them as well?? 


Here is what i can see compared to what's on Figma 👇🏽 

| Your Design  |Figma Design |
| :---:   | :---: | 
| ![erosbad](https://user-images.githubusercontent.com/99246359/206850464-453267d6-e10a-4575-bad9-595f435e804d.PNG) |![erosgd](https://user-images.githubusercontent.com/99246359/206850470-d29d50c0-5f64-46a2-a12f-289b8303c50f.PNG)  |